README.md

```diff
@@ -4,7 +4,8 @@ Tracking time is as simple as typing:
 ```
 jirawatch track ISSUE-873
 ```
-After that, jirawatch will start tracking your time and once you're done, a simple SIGINT (Ctrl + c) will stop tracking and save it to your Jira worklogs.
+After that, jirawatch will start tracking your time and, once you're done, you can press `Ctrl-c` to stop the time tracking and allow jirawatch to save it into the Jira worklogs of the related issue.
+If you want to make a pause, pressing the `Ctrl-p` will stop the time tracking and pressing it again will restart the timer.
```

Comment:
You don't press keys, you hit keys 😄 (i.e use hit instead of press eg. "hit `Ctrl+c`" )

Moreover, the last sentence should be:
If you need to stop tracking time, just hit `Ctrl-p` and then hit it again to resume

Reply:
When you type on your keyboard you **press** the keys! So I think that the `hit` keyword doesn't fit well in this context: what about `smash`? (just kidding)
I would keep `press`

About the Readme, I will change it as you like but I will keep using `press`

Reply:
It seems [https://english.stackexchange.com/questions/152777/do-you-hit-or-press-a-button](both terms are valid), so just keep `press` 😄 
Remember to change the last sentence as I wrote before!

Reply:
I already did with the last commit